lib/dentaku/ast/case.rb

```diff
@@ -1,16 +1,6 @@
-require_relative 'case/case_conditional'
-require_relative 'case/case_when'
-require_relative 'case/case_then'
-require_relative 'case/case_switch_variable'
-require_relative 'case/case_else'
```

Comment:
How are we able to delete all these case child nodes?

Reply:
Leftover from the parser